src/rupali/Shop.java

```diff
@@ -0,0 +1,105 @@
+package rupali;
+
+public class Shop {
+	
+	int stockofMaggie=50;
+	int stockofDosa=43;
+	int stockofOilPouches=39;
+	int stockofPanipuri=43;
+	int stockofmasala=73;
+	
+	void requestedQ(boolean reqType,int maggieq, int dosaQ, int oilQ, int panipuriQ, int masalaQ ){ 
+		if(maggieq==0) 
+			System.out.println("Sorry maggie running out of stock");
+		else 
+			  if(stockofMaggie>=maggieq) {
+				System.out.println("We served your request for maggie");
+				stockofMaggie=stockofMaggie-maggieq;
+			  }
+				else if(reqType==true) {	
```

Comment:
you can write only reqType not to compare with == true(rememberer if condition work on boolean)